packages/timeline-state-resolver/src/integrations/quantel/diff.ts

```diff
@@ -104,6 +105,36 @@ export function diffStates(
 		if (a.command.type !== QuantelCommandType.RELEASEPORT && b.command.type === QuantelCommandType.RELEASEPORT) return 1
 		return 0
 	})
+
+	// If we run any play-command, we will need to cancel any delayed/waiting out-transitions for that port:
+	const portIdsToCancelWaiting: Set<string> = new Set()
+	for (const cmd of allCommands) {
+		if (
+			(cmd.command.type === QuantelCommandType.PLAYCLIP ||
+				cmd.command.type === QuantelCommandType.PAUSECLIP ||
+				cmd.command.type === QuantelCommandType.CLEARCLIP ||
+				cmd.command.type === QuantelCommandType.RELEASEPORT) &&
+			!cmd.command.fromLookahead
```

Comment:
I think this should be looking at the `port.notOnAir` property instead like it does in https://github.com/nrkno/sofie-timeline-state-resolver/blob/ff443bb9260e05288c0c7c96950233ca79d8a679/packages/timeline-state-resolver/src/integrations/quantel/diff.ts#L204

However I can see that in the original release50 implementation this wasn't the case either (but it wasn't triggering for clear and release there either)